### PR TITLE
Compose app scaffold

### PR DESCRIPTION
**Pull request by Elforama**
Setup basic navigation and screens in compose.

Language Selection
Org Selection
Email/Phone Input
Name Input
Dashboard

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.koin.core.context.GlobalContext
+import timber.log.Timber
+
+
+@Composable
+fun SplashScreen(preferencesMigrator: PreferencesMigrator, navController: NavController?) {
+    LaunchedEffect(true) {
+        Timber.tag("BuildVariant").d("build variant: ${BuildConfig.BUILD_TYPE}")
+        preferencesMigrator.migrateIfNeeded()
+        delay(100)
```
Comment (thenotoriousrog) [inline app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt:26]: Question: What does this delay do? I've never seen this before. Curious on what it does!

**Review by thenotoriousrog — APPROVED** [review]
Looks good!

File: app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt (hunk 1 above)
Comment (Elforama) [reply app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt:26]: It's just to show the splash screen for 1 second, but it's .1 second while building out the app

Comment (Elforama) [reply app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt:26]: It pauses the coroutine.

**Pull request merged**